models/stg_linkedin_ads__ad_analytics_by_campaign.sql

```diff
@@ -0,0 +1,47 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_linkedin_ads__ad_analytics_by_campaign_tmp') }}
+),
+
+macro as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_linkedin_ads__ad_analytics_by_campaign_tmp')),
+                staging_columns=get_ad_analytics_by_campaign_columns()
+            )
+        }}
+    from base
+),
+
+fields as (
+    
+    select 
+        cast(day as {{ dbt_utils.type_timestamp() }}) as date_day,
+        campaign_id,
+        clicks,
+        impressions,
+        {% if var('linkedin_ads__use_local_currency', false) %}
+        cost_in_local_currency as cost
+        {% else %}
+        cost_in_usd as cost
```

Comment:
Is there a difference between using `cost` vs `spend`? We use spend consistently across the packages and curious if we should adjust here. What are your thoughts?

Reply:
Same comment for other models.

Reply:
not that i can tell -- i think we just continued to use the source data column name

i do wonder if it would be annoying for someone just using linkedin (and not other ad platforms) for us to rename `cost` to `spend`. however, i don't know how many people are just adversing on linkedin, and it might be more inconvenient for users of the ad reporting package to not rename it in the upstream linkedin package